docs/docs/optimizing-performance.md

```diff
@@ -0,0 +1,118 @@
+---
+id: optimizing-performance
+title: Optimizing Performance
+permalink: docs/optimizing-performance.html
+---
+
+Internally, React uses several clever techniques to minimize the number of costly DOM operations required to update the UI. For many applications, using React will lead to a fast user interface without doing much work to specifically optimize for performance. Nevertheless, there are several ways you can speed up your React application.
+
+## Use The Production Build
+
+If you're benchmarking or experiencing performance problems in your React apps, make sure you're testing with the [minified production build](/react/downloads.html). The development build includes extra warnings that are helpful when building your apps, but it is slower due to the extra bookkeeping it does.
```

Comment:
This PR does also change the "installation" page to briefly mention the production build. The webpack code here is helpful though... I just added this content inline, I think this works. I kept the .min.js links on the installation page because they seem logical there too.